sigstore/_internal/rekor/__init__.py

```diff
@@ -27,7 +27,11 @@
 from .checkpoint import SignedCheckpoint
 from .client import RekorClient
 
-__all__ = ["RekorClient", "SignedCheckpoint"]
+__all__ = [
+    "RekorClient",
+    "SignedCheckpoint",
+    "_hashedrekord_from_parts",
```

Comment:
I suppose at this point it should be just `hashedrekord_from_parts()` without underscore. Since `rekor` is private, it does not seem important tough...

Reply:
Yeah, it probably should since it's entirely private APIs. I'll iterate on this with the follow-up PRs for refactoring.